tutorials/ui/size_and_anchors.rst

```diff
@@ -17,37 +15,53 @@ and mobile phones have different resolutions and aspect ratios.
+(in case there is no parent control) the viewport. Anchors adjust where the
+margin distances are relative *to*. We can think of them as a percentage of the
+parent's size, represented by a value between 0 and 1.0.
 
-.. image:: img/anchors.png
+:ref:`Containers <doc_gui_containers>` are used to compose the layout of control
+scenes, offering powerful layout and sizing functionality. The MarginContainer
+is where we typically set margins for its child controls. When using containers,
+controls will inherit the layout properties of their parent, removing the need
+to modify the anchors of a control.
+
+Layout Presets
+--------------
 
-This is done by editing the *margin* properties of controls. Each
-control has four margins: left, right, bottom, and top, which correspond
-to the respective edges of the control. By default, all of
-them represent a distance in pixels relative to the top-left corner of
-the parent control or (in case there is no parent control) the viewport.
+Each control has an individual anchor that can be adjusted from the beginning
+to the end of the parent. So the vertical (top, bottom) anchors adjust from 0
+(top of parent) to 1.0 (bottom of parent) with 0.5 being the center. The
+horizontal (left, right) anchors similarly adjust from left to right of the
+parent.
 
-.. image:: img/margin.png
+Instead of manually setting the *anchor* properties of a control, you can use
+the toolbar's Layout menu, above the viewport. Besides centering, it gives you
+many options to align and resize control nodes. These will automatically set
+the *anchor* properties. 
 
-So to make the control wider you can make the right margin larger and/or
-make the left margin smaller. This lets you set the exact placement
-and shape of the control.
+Note that changing the layout preset will overwrite the position and size of
+the control. Set to Current Ratio will automatically adjust the control's anchor
+points and offsets to match the current size and position of the control.
 
-The *anchor* properties adjust where the margin distances are relative *to*.
-Each margin has an individual anchor that can be adjusted from the
-beginning to the end of the parent. So the vertical (top, bottom) anchors
-adjust from 0 (top of parent) to 1.0 (bottom of parent) with 0.5 being
-the center, and the control margins will be placed relative to that
-point. The horizontal (left, right) anchors similarly adjust from left to
-right of the parent.
+.. image:: img/anchors_dropdown_menu.webp
 
-Note that when you wish the edge of a control to be above or left of the
-anchor point, you must change the margin value to be negative.
+Custom Anchors
+--------------
 
-For example: when horizontal anchors are changed to 1, the margin values
-become relative to the top-right corner of the parent control or viewport.
+The *anchor* properties can be set manually using the custom anchors preset
+option. Here we can change the exact placement and shape of the control
```

Comment:
It's strange that only at this point in the article we explain that you can set anchors manually, when we already explained that, hey, you don't have to. I don't think this flows well.

We also skip the entire Layout section of the Inspector, which is likely the main tool everyone would use, and not the custom anchors or the toolbar. If we go from the Inspector side of things, we can gradually explain that you can position the node directly, then by using layout presets and anchor presets, and then manually. And then mention the toolbar.